scripts/add-cluster.sh

```diff
@@ -147,7 +149,7 @@ fi
 
 oc create secret generic ${SECRET_NAME} --from-literal=token="${SA_TOKEN}" --from-literal=kubeconfig="apiVersion: v1
 clusters:
-- cluster:
+- cluster:${INSECURE_FIELD}
```

Comment:
I believe it will go to the next line but will the indentation be ok ?

Reply:
the value is defined as:
```
"
    insecure-skip-tls-verify: true"
```
which means it already contains the new line - I wanted to avoid the situation when there is an extra empty line for the secured connections.

Reply:
Right I saw the new line was there but it wasn't evident how the final indentation would look like.